cpp/src/io/avro/avro_gpu.cu

```diff
@@ -24,6 +24,212 @@ namespace gpu {
 #define NWARPS                  16
 #define MAX_SHARED_SCHEMA_LEN   1000
 
+/*
+ * Avro varint encoding - see
+ * https://avro.apache.org/docs/1.2.0/spec.html#binary_encoding
+ */
+static inline int64_t __device__ avro_decode_varint(const uint8_t *&cur, const uint8_t *end) {
```

Comment:
```suggestion
static inline int64_t __device__ avro_decode_zigzag_varint(const uint8_t *&cur, const uint8_t *end) {
```
Maybe make the name a little more specific to help when reading through the algorithm

Reply:
Done